backend/services/Makefile.common

```diff
@@ -78,12 +78,12 @@ docker-publish:
 endif
 
 $(binfile): $(GOFILES)
-	# TODO: Add -ldflags "-X ...$(Version)"
 	env CGO_ENABLED=$(CGO_ENABLED) \
 		GOOS=$(GOOS) \
 		GOARCH=$(GOARCH) \
 		go build -o $(binfile) \
 			-ldflags '$(LDFLAGS)' \
+			-ldflags '-X github.com/mendersoftware/mender-server/pkg/version.version=$(VERSION)' \
```

Comment:
just to double check, VERSION come from where here?

Reply:
It's almost at the top of the file: https://github.com/mendersoftware/mender-server/blob/0701064f0ee1784feea9178fc11d08c033805cca/backend/services/Makefile.common#L14